src/test/regress/sql/pg16.sql

```diff
@@ -591,28 +591,8 @@ REVOKE role1 FROM role2;
 RESET citus.log_remote_commands;
 RESET citus.grep_remote_commands;
 
---
--- PG16 added new options to GRANT ROLE
--- inherit: https://github.com/postgres/postgres/commit/e3ce2de
--- set: https://github.com/postgres/postgres/commit/3d14e17
--- We don't propagate for now in Citus
---
```

Comment:
Let's not remove these comments, but instead update them to say that: "We now propagate these options in Citus"